src/layout/page-layout.scss

```
@@ -267,31 +270,34 @@ $Layout-responsive-variant-max-breakpoint: 'md' !default;
       &::after {
         @include Layout-line-divider;
 
+        // stylelint-disable-next-line primer/spacing
         bottom: calc(#{$border-width * -1} - var(--Layout-row-gap));
       }
     }
 
     .PageLayout-region--dividerNarrow-filled-before {
       position: relative;
       // stylelint-disable-next-line primer/spacing
-      margin-top: calc(#{$spacer-2} + var(--Layout-row-gap));
+      margin-top: calc(var(--base-size-8) + var(--Layout-row-gap));
 
       &::after {
         @include Layout-filled-divider;
 
-        top: calc(#{$spacer-2 * -1} - var(--Layout-row-gap));
+        // stylelint-disable-next-line primer/spacing
+        top: calc(var(--base-size-8) * -1 - var(--Layout-row-gap));
```

```suggestion
        top: calc(calc(var(--base-size-8) * -1) - var(--Layout-row-gap));
```

Maybe? I'm not sure if this is needed 🤔

---

It seems to work either way, so I'll add